### PR TITLE
Navbar colours

### PR DESCRIPTION
**Pull request by sazzamac**

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -22,8 +22,9 @@ const MyTheme = {
   ...DefaultTheme,
   colors: {
     ...DefaultTheme.colors,
-    background: 'rgb(252,252,252)',
-    card: 'rgb(252,252,252)',
+    background: '#091954',
```
Comment (KejPat) [inline App.js:25]: Think the background colour should be #0A0A2A, a different background colour looks odd

File: App.js [hunk 2]
```diff
@@ -56,15 +57,16 @@ componentDidMount() {
               elevation: 0,
```
Comment (KejPat) [inline App.js:57]: also can you take out the backgroundColor in the stack navigator, I actually don't know why I put it there but it seems redundant

File: App.js (hunk 1 above)
Comment (sazzamac) [reply App.js:25]: yeah the dark navy is nice!

**Review by Sajni94 — APPROVED** [review]
looking gooood!

**Review by KejPat — APPROVED** [review]

**Pull request merged**